sys/kern/kern_sig.c

```diff
@@ -3130,16 +3129,7 @@ issignal(struct thread *td)
 				printf("Process (pid %lu) got signal %d\n",
 					(u_long)p->p_pid, sig);
 #endif
-				/*
-				 * Deliver signals that would cause a sandbox
-				 * unwind (fatal signals) to init to avoid an
-				 * inifinite loop on boot if we miscompile init.
-				 */
-				if (p->p_pid == 1 && (prop & SIGPROP_SBUNWIND) != 0)
```

Comment:
I do wonder if we want to keep this bit, possibly upstream it, using SIGPROP_CORE... squashing all signals is rather horrible (though rather less commonly hit for us than it was 5 years ago)

Reply:
Well, you wouldn't want `kill -QUIT 1` to turn into a panic out of the box I think.

Reply:
I mean, play stupid games, win stupid prizes... is there a legitimate reason you'd ever send a SIGPROP_CORE signal to init?

Reply:
That argument might cut both ways though that any SIGPROP_CORE signal is illegitimate and hence best ignored.  That said, there is an XXX comment just earlier about not ignoring SIGSEGV's for init.  I'm not really sure which is worse, init running in an infinite loop because it keeps retrying a faulting instruction or the kernel panicking.  Probably there are some cases where you can still log in to a host with a looping init and shut processes down to do a more orderly reboot to recover.  For CheriBSD though I'd mostly just like to reduce our diff.